app/components/post_list/post_list.js

```diff
@@ -140,17 +171,33 @@ export default class PostList extends PureComponent {
         }
     };
 
+    measureItem = (index, height) => {
+        this.itemMeasurements[index] = height;
+        if (this.props.postIds.length === Object.values(this.itemMeasurements).length) {
+            if (this.newMessagesIndex !== -1 && !this.newMessageScrolledTo) {
+                this.setState({
+                    scrollToMessage: true
+                });
+            }
+        }
+    }
+
     renderItem = ({item, index}) => {
         if (item === START_OF_NEW_MESSAGES) {
             this.newMessagesIndex = index;
+            this.moreNewMessages = this.props.postIds.length === index + 2;
```

Comment:
I'm not sure I understand how you get this value

Reply:
So we pull in 15 posts and a date header for the initial post display. If there is a new message `START_OF_NEW_MESSAGES` is added as well making our overall `postIds.length` 17. So if the new message index is 15 which would technically be the end of the post list, we have to add 2 to account for the extra 2 items that are added on.

Reply:
Oh okay, I think I understand that. Could you add a comment explaining this so we can remember what this does when we come across this code in the future?